index.js

```diff
@@ -188,7 +164,7 @@ function sendNotification(endpoint, params) {
         options.headers['Authorization'] = 'Bearer ' + jwt;
         var key = 'p256ecdsa=' + urlBase64.encode(vapid.publicKey);
         if (options.headers['Crypto-Key']) {
-          options.headers['Crypto-Key'] += ',' + key;
+          options.headers['Crypto-Key'] += ';' + key;
```

Comment:
Shouldn't this be a comma?
At least this is what Autopush is expecting: https://github.com/mozilla-services/autopush/blob/f546ed78585976d33534595dafcef401c118ea43/autopush/crypto_key.py and https://github.com/mozilla-services/autopush/blob/4361bbe07ee4b962622a3c16ddf632405a599158/autopush/tests/test_cryptokey.py